mcp/health-tool-mcp/server.py

```
@@ -0,0 +1,1262 @@
+            return _log_output_and_return(
+                tool_name="fhirPackageGeneration",
+                request_id=_request_id,
+                caller=_caller,
+                response_str=format_error_output(
+                    "fhirPackageGeneration",
+                    "Empty directory",
+                    f"Empty definitions at: {fhir_spec_directory}. {setup_hint}"
+                ),
+                start_time=_start_time,
+            )
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)